### PR TITLE
[BOOST-652] Implement skeleton for azure-infrastructure

### PR DESCRIPTION
**Pull request by MarcAstr0**
## Description
Implement skeleton for azure-infra package

## Changes
Tested locally so that when running a boost deploy we will deploy a resource group and a storage account.

## Checks
- [X] Project Builds
- [X] Project passes tests and checks
- [ ] Updated documentation accordingly at [the docs repo](https://github.com/boostercloud/docs)
- [ ] Added a link pointing to the documentation PR in this PR

### CODE REVIEW

**Comment by MarcAstr0** [comment]
> Itis weird, check the linters because they failed

It fails because it can't bootstrap the project:
```
npx: installed 753 in 36.985s
5
lerna notice cli v3.22.1
6
lerna info ci enabled
7
lerna info bootstrap root only
8
yarn install v1.22.4
9
[1/4] Resolving packages...
10
error Couldn't find any versions for "@boostercloud/framework-provider-azure" that matches "^0.3.3"
11
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
12
warning workspace-aggregator-3075625d-b301-46b3-ac47-b6de6e6ee8d5 > @boostercloud/framework-provider-azure-infrastructure > ms-rest-azure > adal-node > request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
13
lerna ERR! yarn install --mutex network:42424 --non-interactive exited 1 in 'root'
14
lerna ERR! yarn install --mutex network:42424 --non-interactive exited 1 in 'root'
15
##[error]Process completed with exit code 1.
```

File: packages/framework-provider-azure-infrastructure/package.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  "files": [
+    "dist"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/boostercloud/booster.git"
+  },
+  "dependencies": {
+    "@boostercloud/framework-types": "^0.3.3",
+    "@boostercloud/framework-provider-azure": "^0.3.3",
```
Comment (rdiaz82) [inline packages/framework-provider-azure-infrastructure/package.json:24]: this line should be 0.4.0

File: packages/framework-provider-azure-infrastructure/package.json [hunk 2]
```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@boostercloud/framework-provider-azure-infrastructure",
+  "version": "0.3.3",
```
Comment (rdiaz82) [inline packages/framework-provider-azure-infrastructure/package.json:3]: this should be also 0.4.0

**Comment by rdiaz82** [comment]
bot: integration

**Pull request merged**